pkg/api/bytes_test.go

```diff
@@ -57,6 +58,14 @@ func TestBytes(t *testing.T) {
 		}
 	})
 
+	t.Run("download-with-targets", func(t *testing.T) {
+		resp := request(t, client, http.MethodGet, resource+"/"+expHash+"?targets="+targets, nil, http.StatusOK)
+
+		if resp.Header.Get("Targets") != targets {
```

Comment:
this should be extracted to some package level const that defines the `Targets` header. also, `Targets` in the sphere of `HTTP` is very general. it would be better to have some application level header, such as `swarm-recovery-targets` or `swarm-repair-targets` or `swarm-targets` etc you get the point

Reply:
I went for `swarm-recovery-targets`
Very good point indeed.
Fixed